### PR TITLE
Blog: "Open Source Development" post

### LINKED ISSUE #190: Add open source blog post
Blog post copy can be found in [Google Docs](https://docs.google.com/document/d/12EZEh-0JkI0VZqOn7i0KAGQ_zfc8Cp5mE1Lw58lNDhU/edit)

### PR DESCRIPTION
**Pull request by AnthonyRollins**
Closes #190

**This PR**
- Adds the blog post file for Open Source development
- Adds 3 new images to the blog assets folder

### CODE REVIEW

File: src/_posts/2024-04-12-open-source-development.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Our goal at Compiler is simple - to help government agencies meet their strategic goals and better serve communities. We do that by providing customized solutions and impactful consulting services to improve IT workflows within organizations. One key component of this is working on code that is open source. We feel strongly that software and code developed using tax dollars should be produced as open source wherever possible, as this increases transparency, reduces opportunities for vendor lock-in, and delivers better and more cost-effective results to the public.
+
+<figure>
+    <img src="/assets/blog/2024/2024-4-metro-k-line.png" alt="A photograph of the Metro K line light rail train on the track in Los Angeles" />
+    <figcaption>Metro K line in Los Angeles</figcaption>
+</figure>
+
+Some of the benefits of open-source software development include:
+
```
Comment (laneymangan) [inline src/_posts/2024-04-12-open-source-development.md:27]: There's an oddly large amount of space between the line here and the first bullet point, I'm not sure if there's any way to fix that?

**Review by laneymangan — APPROVED** [review]
Made one minor comment but otherwise I think it looks great!

File: src/_posts/2024-04-12-open-source-development.md (hunk 1 above)
Comment (thekaveman) [reply src/_posts/2024-04-12-open-source-development.md:27]: This is a problem with the styling, nothing that can be fixed in the content itself.

**Review by thekaveman — APPROVED** [review]

**Comment by AnthonyRollins** [comment]
@laneymangan @thekaveman, even though some stylings need to be fixed, are we okay with me merging this?

**Comment by laneymangan** [comment]
I'd say so! Only thing is making sure we're good with the pictures re: [Vyki's slack message](https://compilerla.slack.com/archives/C05L2B0SG06/p1713196911222619?thread_ts=1712936795.284769&cid=C05L2B0SG06): `just checking for the photos the IP is ok, they’re CC0 or similar? Ok for unattributed commercial use?`

**Pull request merged**